File: A1/A1-020.cpp
#include <iostream>
using namespace std;

int main()
{
    int n[4];

    // input
    cin>> n[0]>> n[1]>> n[2]>> n[3];

    // process
    if (n[0] < n[1] && n[1] < n[2])
    {
        // output
        cout<< "increasing";
    }
    else if (n[0] > n[1] && n[1] > n[2])
    {
        cout<< "decreasing";
    }
    else
    {
        cout<< "neither";
    }
}

File: Python/A1/A1-020.py
# Author: Korawit Suporn
# Date  : 23 Apr. 2025
# Editor: Vim

arr = [int(input()) for _ in range(3)]

if arr[0] > arr[1] and arr[1] > arr[2]:
    print("decreasing")
elif arr[2] > arr[1] and arr[1] > arr[0]:
    print("increasing")
else:
    print("neither")
